docker-compose.ci.yml

```diff
@@ -0,0 +1,60 @@
+version: '3'
+
+services:
+    ci:
+        build:
+            context: .
+            dockerfile: Dockerfile.ci
+            args:
+                image_tag: ${IMAGE_TAG}
+        image: elifesciences/journal_ci:${IMAGE_TAG}
+        volumes:
+            - ./.docker/parameters.yml:/srv/journal/app/config/parameters.yml
+            - fixtures:/srv/journal/var/fixtures
+            - tests:/srv/journal/test
+            - vendor:/srv/journal/vendor
+        environment:
+            - APP_ENV=ci
+        depends_on:
+            - cli
+            - selenium
+    cli:
+        environment:
+            - APP_ENV=ci
+        command: sleep infinity
+    composer:
+        build:
+            args:
+                composer_dev_arg: --no-dev
+    composer-dev:
+        build:
+            context: .
+            dockerfile: Dockerfile.composer
+            args:
+                composer_dev_arg:
+        image: elifesciences/journal_composer_dev:${IMAGE_TAG}
+        command: /bin/bash
+    fpm:
+        volumes:
+            - fixtures:/srv/journal/var/fixtures
+            - tests:/srv/journal/test
```

Comment:
so the reason for the volumes for `tests` and `vendor` is that `fpm` is mounting them, probably because there is some hidden dependency? Related to canned responses? This is a divergence between the image that will be published for use in production and what is being tested, I'll look into it to understand it better.

Reply:
Or we could try and avoid the service decoration.

Reply:
Looking at the two classes it seems reasonable to use them in multiple environments and just during the test suite, so I second the motion to move them into `src`

Reply:
Hmm, it has a dependency on `elife/api-validator` too, which is currently `require-dev`. Looking at removing these services if the classes don't exist, as they're not actually needed to read the fixtures.